site/public/_about.md

```diff
@@ -16,3 +16,24 @@ page with,
 The `All Versions` link on each distribution shows directory listings for all the available versions, and a `versions` file ([example](http://dist.ipfs.io/go-ipfs/versions)). This file can be used by tools, such as [ipfs-update](#ipfs-update), to find all the available versions and download the latest.
 
 The directory listing of each version ([example](http://dist.ipfs.io/go-ipfs/v0.3.11)) has all the platform archives (`.zip` or `.tar.gz`), a `README.md` and a `dist.json` which describe the release for humans and machines. It is meant to be easily consumed and used by tools.
+
+##### Code Signing
+
+All releases are signed using [OpenPGP](http://www.openpgp.org/). You can verify this by running
+
+```bash
+$ gpg --verify go-ipfs.tar.gz.asc go-ipfs.tar.gz
+```
+
+You will need to download the public key of the release managers, which are currently,
+
+* Friedel Ziegelmayer <dignifiedquire@gmail.com> [`E2C5 3DFE 7CBA 9864 38B9  88D9 0741 3B8A 27F5 0659`](https://pgp.mit.edu/pks/lookup?search=0xE2C53DFE7CBA986438B988D907413B8A27F50659&op=index&fingerprint=on&exact=on).
+
+The command for this is
+
+```bash
+$ gpg --keyserver pgpkeys.mit.edu --recv-key <keyid>
+```
```

Comment:
Instead of doing this, which means that user has to import the key and will get warning that files are signed by key he doesn't trust. We could use `gpgv`/`gpgv2` and custom keyring file. See: http://unix.stackexchange.com/a/78476